test/spanner/bundle/validator_test.exs

```diff
@@ -19,6 +19,10 @@ defmodule Spanner.Bundle.ValidatorTest do
     assert validate("no_templates") == :ok
   end
 
+  test "commands can only have 'once' or 'multiple' execution types" do
```

Comment:
Testing that `"once"` and `"multiple"` are valid would be good, but this test doesn't do that, so it should be renamed.

Can you write tests that accept YAML from a heredoc instead of a separate file? That would allow fixture data to be close to the tests, and would allow you to generate tests automatically like

```elixir
Enum.each(["once", "multiple"], fn(execution) ->
  test "an execution value of `#{execution}` is valid" do
    #... assertion with YAML in a heredoc
  end
end)
```